ui/assets/styles/general/tool.styles.js

```diff
@@ -148,7 +148,7 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   detailsContainer: {
-    height: '30rem',
+    height: '44rem',
```

Comment:
@sudhanshutech this was not the ask, the ask here is to make sure it always take the full page, you should use the `vh` unit here, also make sure if height increases we maintain the scroll bar otherwise if height increases and scrollbar hides user will not able to trigger new fetch request on scroll

Reply:
Thank you for highlighting the importance of using the vh unit @Yashsharma1911 . While I was already aware of the ask, your reminder was valuable. I have now made the necessary adjustments to use vh instead of rem to meet this requirement. Additionally, I've ensured that if the height increases, the scroll bar will be maintained to allow users to trigger new fetch requests on scroll.

Reply:
Okay I'll test

Reply:
have you tested @Yashsharma1911 ?